plugins/foreman_virt_who_configure/0.1/index.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: plugin
+title: Foreman Virt Who Configure 0.1 Manual
+version: 0.1
+---
+
+# 1. {{ page.title }}
+
+This plugin make [virt-who](https://github.com/virt-who/virt-who) configuration easy. It provides simple UI for obtaining required information about the desired virt-who configuration, such as hypervisor credentials, check interval and similar. After the information is gathered, it provides a configuration script that is used to install and configure the virt-who instance. All incoming reports from such virt-who instance are tracked and monitored by the plugin.  
+
+# 2. Installation
+
+This plugins requires [Katello](https://theforeman.org/plugins/katello/index.html) to be installed in your Foreman instance. If you have it, simply install a package with the plugin (rpm only), run migrations and seed by running
+
+    yum install rubygem-foreman_virt_who_configure
+    foreman-rake db:migrate
+    foreman-rake db:seed
+
+Plugin package lives in [Katello repositories](https://fedorapeople.org/groups/katello/releases/yum/).
+
+If installed foreman from git, you can simply add foreman-virt_who_configure gem as one of the dependencies. Don't forget to run migrations and seed afterwards and obviously restart the Foreman. You still need to have Katello installed.
+
+Note for Satellite 6 users, the plugin is installed out of the box and no extra action is required.  
+
+# 3. Configuration
+
+There's nothing to configure for this plugin yet. See the next chapter.
+
+# 4. Usage (features description)
+
+The plugin adds a new link "Virt-who configurations" under Infrastructure menu. This page lists all virt-who configurations that were created in Foreman. To create a new configuration, click Create Config button. Once provide all information, the configuration is registered and you can deploy it. Each configuration provides a configuration script that you find after clicking on its name. The configuration script should be run on host where you want to deploy virt-who instance at, e.g. the same host where Foreman runs. Existing virt-who configurations can be deleted when no longer required. Plugin also provides monitoring widget for dashboard. Please see chapters below for more details on every feature.
```

Comment:
Once *you* provide all information ...

The configuration script should be run on *a* host where ...